support/modules/file-install

```diff
@@ -21,19 +23,35 @@ case "$STATE" in
 
     ArtifactInstall)
         dest_dir=$(cat $dest_dir_file)
-        [[ -d $dest_dir ]] || install -d $dest_dir
-        tar -cf ${prev_files_tar} -C ${dest_dir} .
-        rm -rf ${dest_dir}
-        mkdir -p ${dest_dir}
+        single_file=$(cat $single_file_file)
+        test -z "$dest_dir" -o -z "$single_file" && exit 1
```

Comment:
The `exit 1` are very silent and would prefer some text here. Otherwise the log from a failed update would be useless.

Reply:
This should never happen if you create the artifact with the provided generator tool.

But you are right about the silent part and I would prefer some text also :) I'll fix.

Reply:
Done